packages/rebber-plugins/__tests__/__snapshots__/rebber.test.js.snap

```diff
@@ -255,8 +259,11 @@ a paragraph\\\\footnote[l9c12o138]{\\\\label{footnote:l9c12o138} footnoteRawPar
 
 exports[`gridTable 1`] = `
 "\\\\begin{longtabu}{|p{\\\\linewidth / 3}|p{\\\\linewidth / 3}|p{\\\\linewidth / 3}|} \\\\hline
```

Comment:
Not that I'm against it, but what is the technical reason for maintaining this construct while `X[-1]` should do it ? (if you want three columns that have the same size but do not want to cares about the number of columns, you can also consider `X[1]`, which will give column of equal sizes no matter the number).

Reply:
In fact as your issue was only for normal tables I thought you wanted the gridtable behaviour to be kept. that's why you have the old syntax here.

Reply:
Did you manage to check this behaviour?